docs/source/getting-started/testing.rst

```diff
@@ -0,0 +1,134 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Testing Your App
+****************
+
+When developing your application, you will want to test certain workflows through the UI.
```

Comment:
We should use the phrase "end-to-end tests" here. And maybe something like "all the way from the UI to the ledger and back".